### ISSUE TITLE
Windows 10 error

### ISSUE THREAD

**Issue by ran-j** [enziop/mixamo_converter#67]
I'm using blender 2.7 on windows 10 and it gives me a following error:

![Screen Shot 04-25-20 at 02 20 PM](https://user-images.githubusercontent.com/17410205/80286189-50965280-8700-11ea-9e11-ea33f3641049.PNG)

**Comment by DJL3G3ND** [comment]
YES, THANK YOU, finally someone else who has this problem, its been driving me insane for the last like four hours trying to find a fix and Ive found nothing, at least Im glad to see someone else posting it here and hopefully we can get an answer. I also tried this on my old windows 8 laptop and got the same thing

**Comment by ran-j** [comment]
kkkkk me too, and I tried on 2 windows 10 pcs and got the same error

**Comment by Steblak** [comment]
Yes, there is this mistake, I already thought that I had such a 
![image](https://user-images.githubusercontent.com/38109041/80298518-bbde3400-8795-11ea-9219-f1af1ebdd4a8.png)

**Comment by ran-j** [comment]
On blender 2.8 kinda works

![image](https://user-images.githubusercontent.com/17410205/80315202-712bde80-87cc-11ea-95d5-aaeff55b82a9.png)

**Comment by enziop** [comment]
for blender 2.7 you have to use the blender 2.7 branch
![image](https://user-images.githubusercontent.com/21155833/80406999-35c10b00-88c5-11ea-978c-fb8c5c3fcd10.png)
but i recommend to use blender28 as some new fixes and features will not be put in the blender 2.7 version
please let me know if that was the problem
